test/objspace/test_ractor.rb

```diff
@@ -0,0 +1,16 @@
+require "test/unit"
+require "objspace"
+
+class TestObjSpaceRactor < Test::Unit::TestCase
+    def test_tracing_does_not_crash
+        ObjectSpace.trace_object_allocations do
+            r = Ractor.new do
```

Comment:
Maybe we need to ensure this Ractor is dead at the end of the test? (just a stab in the dark)